src/components/QrCodeForm.vue

```diff
@@ -0,0 +1,208 @@
+<script setup lang="ts">
+import { pix } from "../utils/QrCodePix";
+import { ref } from "vue";
+import { debounce } from "@/utils/debounce";
+import CustomButton from "./CustomButton.vue";
+import api from "../services/index";
+
+const props = defineProps({
+  pixTarget: String,
+  tokenValue: Number,
+});
+
+console.log(props.tokenValue);
+console.log(props.pixTarget);
+
+const qrCode = ref<string>("");
+const qrCodePayload = ref<string>("");
+const pixQrCode = pix({
+  pixKey: props.pixTarget ?? "",
+  value: props.tokenValue,
+});
+const pixIsValid = ref<number>(0);
+
+pixQrCode.base64QrCode().then((code: string) => {
+  qrCode.value = code;
+});
+
+qrCodePayload.value = pixQrCode.payload();
+
+const handleInputEvent = (event: any) => {
+  const { value } = event.target;
+
+  validatePix(value);
+};
+
+const validatePix = async (value: any) => {
+  console.log(value);
+  if(value == ''){
+    pixIsValid.value = 0;
+    return;
+  }
+  var sellerPixKey = props.pixTarget;
+  var transactionValue = props.tokenValue;
+
+  if(sellerPixKey && transactionValue){
+    var body_req = {
+      e2e_id: value,
+      pix_key: sellerPixKey,
+      pix_value: transactionValue,
+    };
+    var resp = await api.post("http://localhost:8000/validate_pix", body_req)
```

Comment:
Why do you use base_url as "http://localhost:8000" in api axios declaration and you use absolute url here again with "http://localhost:8000/validate_pix"?